test/unit/test_sap_cli_domain.py

```
@@ -0,0 +1,134 @@
+        """Test that lowercase input is converted to uppercase"""
+        connection = Connection([
+            Response(text=DOMAIN_ADT_GET_RESPONSE_XML, status_code=200, headers={})
+        ])
+
+        console = BufferConsole()
+        the_cmd = self.domain_read_cmd('begrm')  # lowercase
+        the_cmd.console_factory = lambda: console
+        the_cmd.execute(connection, the_cmd)
+
+        # Parse the output as JSON (default format is ABAP)
+        output = json.loads(console.capout)
```

Flake8 (7.3.0)

[error] local variable 'output' is assigned to but never used

(F841)